src/modules/services_manager_service.rb

```diff
@@ -175,12 +155,10 @@ def read
     #
     # @return [Boolean]
     def reset
-      self.errors = []
-      self.modified = false
+      services.values.each(&:reload)
```

Comment:
as I said, I really dislike this API name. Here it looks like you reload of services, not reset it.

Reply:
Basically, you are just ignoring the changes that the user did. Perhaps replacing 'reload' with 'reset' more meaningful, but I do not like any of those names :) I will adjust it in the yast2.rpm package too.